src/server/config/schema.js

```diff
@@ -31,8 +31,15 @@ module.exports = Joi.object({
     defaultRoute: Joi.string(),
     ssl: Joi.object({
       cert: Joi.string(),
-      key: Joi.string()
-    }).default(),
+      key: Joi.string(),
+      enabled: Joi.boolean().when('cert', {
```

Comment:
That is why I added the `.add('cert', 'key')` bit, it makes sure that `cert` can't be set without `key` and vice-versa. This way the existence of `cert` is sufficient.